src/atoms/Icon/index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+export const iconTypes = ['solid', 'regular', 'light', 'brands']
+export const iconSizes = ['xs', 'sm', 'lg', '2x', '3x', '5x', '7x', '10x']
```

Comment:
consider adding `4x`, `6x`, and `8x` as listed in the [FA sizing scale details](https://fontawesome.com/how-to-use/on-the-web/styling/sizing-icons#scale)